src/views/SelectLevel.vue

```diff
@@ -1,16 +1,19 @@
 <template>
   <div class="small">
     <Logo />
-    <h1 class="title-levels">Уровень сложности</h1>
-    <div class="btns-levels">
-      <button
-      v-for="level of levels"
-      :key="level.id"
-      class="active-btn active-btn-levels"
-      @click="() => setCurrentLevel(level)"
-      >{{level.title}}</button>
-      <a href="https://docs.google.com/forms/d/e/1FAIpQLSdX8WJ8jmMZNloNDKaDJazrZ12qzvAbL58dvOgeHo6oPVyPow/viewform" target="_blank" rel="noopener noreferrer" class="active-btn-levels active-btn">Оставить отзыв</a>
-    </div>
+    <Loader v-if="!levels.length"/>
+    <main v-else>
+      <h1 class="title-levels">Уровень сложности</h1>
+      <div class="btns-levels">
+        <button
+        v-for="level of levels"
+        :key="level.id"
+        class="active-btn active-btn-levels"
+        @click="() => setCurrentLevel(level)"
```

Comment:
а почему просто не вызываешь функцию setCurrentLevel(level)?

Reply:
Перепутала с react.js . Исправлю на: 
```suggestion
        @click="setCurrentLevel(level)"
```